mainnet/.env.example

```diff
@@ -1,4 +1,4 @@
-MAIN_SERVICE_IMAGE=ghcr.io/layr-labs/eigenda/opr-node:release-0.6.1
+MAIN_SERVICE_IMAGE=ghcr.io/layr-labs/eigenda/opr-node:release-0.6.2
```

Comment:
can you also update holesky with this image - now that both should run this

Also can you ad this command too - https://github.com/Layr-Labs/eigenda-operator-setup/blob/master/holesky/run.sh#L65